packages/fluentui/react-northstar/src/components/Dropdown/DropdownSelectedItem.tsx

```diff
@@ -132,57 +135,61 @@ const DropdownSelectedItem: React.FC<WithAsProp<DropdownSelectedItemProps>> &
     },
   });
 
-  const contentElement = Box.create(header, {
+  const contentElement = Box.create(
+    Box.create(header, {
+      defaultProps: () => ({
+        as: 'span',
+        className: dropdownSelectedItemSlotClassNames.header,
+        styles: resolvedStyles.header,
+      }),
+    }),
+    {
+      defaultProps: () => ({
+        className: dropdownSelectedItemSlotClassNames.header,
+        styles: resolvedStyles.content,
+      }),
+    },
+  );
+
+  const iconElement = Box.create(icon, {
     defaultProps: () => ({
-      as: 'span',
-      className: dropdownSelectedItemSlotClassNames.header,
-      styles: resolvedStyles.header,
+      'aria-label': `Remove ${header} from selection.`, // TODO: Extract this in a behaviour.
```

Comment:
how would the user customize the label? Is it through Dropdown's `getA11ySelectionMessage`?

Reply:
Resolving this to focus the discussion on the PR #13410